src/hints/mod.rs

```diff
@@ -12,21 +12,56 @@ mod skip;
 pub use skip::*;
 
 lazy_static! {
+	/// the `HINT_OUTPUT_BUFFER` is a hasmap<Uuid,String> protected from concurrency
+	/// with a RwLock used by `fn test_single_entrypoint` to store the output of the `cairo_run`
+	/// command for each test entrypoint.
+	///
+	/// Before test execution, an Uuid is associated for the test, test is executed
+	/// and `cairo_run` output is captured in the `HINT_OUTPUT_BUFFER`
+	/// # Examples
+	///
+	/// Basic usage:
+	/// ```
+	///	let mut output = String::new();
```

Comment:
this code gets executed in tests

your documentation is executed inside rust tests
you can add a `no_run` flag

/// ```no_run
/// loop {
///     println!("Hello, world");
/// }
/// `

https://doc.rust-lang.org/rustdoc/write-documentation/documentation-tests.html

Reply:
I've used ignore instead.